components/org.wso2.carbon.identity.application.authenticator.biometric/src/main/resources/Artifacts/css/loading1.css

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Why do we need this css file?

Reply:
This is the css file of loading gif used in the wait.jsp page.